lib/nifty_services/errors.rb

```diff
@@ -1,13 +1,22 @@
 module NiftyServices
-  class Error < Exception
+  class Error < RuntimeError
   end
 
   module Errors
-    BaseService::ERROR_RESPONSE_METHODS.each do |error, status|
+    Configuration::ERROR_RESPONSE_STATUS.each do |error, _status|
       class_eval <<-END
-        class #{error.to_s.camelize} < Error
+        class #{error.to_s.camel_case} < Error
         end
       END
     end
+
+    class InvalidUser < Error
+      MESSAGE = 'Invalid User class.
```

Comment:
This will append a new line character(`\n`) to string too, this is side effect which is not desired.